weave/type_handlers/Audio/utils.py

```diff
@@ -0,0 +1,89 @@
+import shutil
+from enum import Enum
+from typing import Any, Optional
+
+try:
+    from pydub import AudioSegment
+
+    has_pydub = True
+except ImportError:
+    AudioSegment = None
+    has_pydub = False
+
+
+class AudioFormat(str, Enum):
+    """
+    These are NOT the list of formats we accept from the user
+    Rather, these are the list of formats we can save to weave servers
+    If we detect that the file is in these formats, we copy it over directly
+    Otherwise, we encode it to one of these formats using ffmpeg (mp3 by default)
+    """
+
+    MP3 = "mp3"
+    WAV = "wav"
+    UNSUPPORTED = "unsupported"
+
+    def __str__(self) -> str:
+        return self.value
```

Comment:
If you are defining this too you may as well just make it a `AudioFormat: Literal["mp3", "wav"]`